week2/ShoppingList/cart.js

```diff
@@ -0,0 +1,289 @@
+const ITEM_LIST_KEY = "itemList";
+//장바구니(cart)페이지 구현 - 로컬 스토리지 활용
+const cartBody = document.getElementById('cartBody');
+
+const storageItems = localStorage.getItem(ITEM_LIST_KEY);
+const parsedItems = JSON.parse(storageItems); //localStorage에서 가져온 아이템
+
+function deleteGrandParent(e)
+{
+  const grandParent = (e.currentTarget.parentNode).parentNode;
```

Comment:
이게 삭제 이벤트가 발생한 tr 태그값을 잡아오기 위한 부분인 것 같은데, 더 간단하게 `closest`라는 속성을 사용해볼 수 있답니다 !
[closest 속성이란?](https://developer.mozilla.org/ko/docs/Web/API/Element/closest)
테이블 아이템을 동적으로 생성할 때 tr에 식별가능한 class명을 부여하고, 해당 class명을 통해서 접근하면 더 편해요!
```html
<tr class="cart-list__tr"></tr>
```

```suggestion
  const removedItemId = e.currentTarget.closest('.cart-list__tr').id;
```
이렇게 한다면 바로 삭제 이벤트가 발생한 아이템의 id값에 접근할 수 있답니다 !

Reply:
오 parentNode로 접근하는 방식은 코드 긴장도가 높았는데, 확실히 좋은 방법인 것 같아요!
바로 적용해보겠습니다. 감사합니다 😊